pkg/agent/metrics/collector.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"strconv"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/ovs/ovsctl"
+)
+
+type ovsMeterStatsCollector struct {
+	ovsCtlClient ovsctl.OVSCtlClient
+}
+
+func NewOVSMeterStatsCollector(ovsCtlClient ovsctl.OVSCtlClient) *ovsMeterStatsCollector {
+	return &ovsMeterStatsCollector{ovsCtlClient}
+}
+
+func (collector *ovsMeterStatsCollector) Run(stopCh <-chan struct{}) {
+	wait.Until(collector.collectMeterStats, time.Second*30, stopCh)
+}
+
+func (collector *ovsMeterStatsCollector) collectMeterStats() {
+	klog.InfoS("Collect meter stats")
+	statsMap, err := collector.ovsCtlClient.GetMeterStats()
+	if err != nil {
+		klog.InfoS("Failed to get meter stats", "error msg", err)
```

Comment:
I think @tnqn pointed out in the past that you should not have spaces in key names for structured log messages.
We could probably use `klog.ErrorS(err, ...)` here anyway.

Reply:
Thanks, will update.